backend/src/file/file.config.ts

```diff
@@ -0,0 +1,25 @@
+import { Inject, Injectable } from '@nestjs/common';
+import {
+  MulterModuleOptions,
+  MulterOptionsFactory,
+} from '@nestjs/platform-express';
+import * as GridFsStorage from 'multer-gridfs-storage';
+
+Injectable();
+export class GridFsMulterConfigService implements MulterOptionsFactory {
+  gridFsStorage: GridFsStorage;
+  constructor(@Inject('DATABASE_CONNECTION') private readonly db: any) {
```

Comment:
please change injection string `'DATABASE_CONNECTION'`  into a variable, may be create `/database/constants.ts` file, then import it to `@inject()`